### DISCUSSION TITLE
How to use windi css plugings in lume?

### DISCUSSION THREAD

**Discussion by officialrajdeepsingh** [lumeland/lume#318] [Q&A] [answered]
I need windi CSS's [official typography plugin](https://windicss.org/plugins/official/typography.html) for my blog but I can find ways how to config with the lume windi CSS plugin. 
I try some thing but cant work
### first try 
```
site.use(windi_css({
  config:{
    plugins:['typography']
  }
}));
```
### second try
```
import typography from 'npm:windicss/plugin/typography';

site.use(windi_css({
  config:{
    plugins:['typography']
  }
}));
```

### third try 
```
import typography from 'windicss/plugin/typography';

site.use(windi_css({
  config:{
    plugins:['typography']
  }
}));
```
I can also create `windi.config.js` file it also not work

**Comment by oscarotero** [comment, marked as answer]
In your examples, you're passing a string as a plugin `plugins:['typography']`, I guess you need to pass the plugin object:

```js
import typography from 'npm:windicss/plugin/typography';

site.use(windi_css({
  config:{
    plugins:[typography]
  }
}));
```
